spec/pluralrules.html

```diff
@@ -53,6 +53,82 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-GetOperands" aoid="GetOperands">
+       <h1>GetOperands (x)</h1>
+       <p>
+         When the GetOperands abstract operation is called with argument _x_ (which must be a finite Number value or a String representation of it), it returns an Object value with the following list of plural operands used by plural rules functions.
+       </p>
+
+       <emu-table id="table-plural-operands">
+         <emu-caption>Plural Rules Operands</emu-caption>
+         <table class="real-table">
+           <thead>
+             <tr>
+               <th>Symbol</th>
+               <th>Value</th>
+             </tr>
+           </thead>
+           <tr>
+             <td>n</td>
+             <td>absolute value of the source number (integer and decimals)</td>
+           </tr>
+           <tr>
+             <td>i</td>
+             <td>integer digits of n.</td>
+           </tr>
+           <tr>
+             <td>v</td>
+             <td>number of visible fraction digits in n, <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>w</td>
+             <td>number of visible fraction digits in n, <i>without</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>f</td>
+             <td>visible fractional digits in n, <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>t</td>
+             <td>visible fractional digits in n, <i>without</i> trailing zeros. </td>
+           </tr>
+         </table>
+       </emu-table>
+
+       <p>
+        The following steps are taken:
+       </p>
+
+       <emu-alg aoid="GetOperands">
+         1. Let _sv_ be ? ToString(_x_).
+         1. If _sv_ starts with *"-"*, then
+           1. Let _sv_ be the substring of _sv_ from position 1, inclusive.
+         1. Let _dp_ be Call(%StringProto_indexOf%, _sv_, « `"."` »).
+         1. Let _n_ be ? ToNumber(_stringValue_).
+         1. Let _i_, _v_, _f_, _t_, _w_ be *0*
+         1. If _dp_ is -1, then
+           1. Set _iv_ to _sv_
+         1. Else,
+           1. Let _iv_ be the substring of _sv_ from position 0, inclusive, to position _dp_, exclusive.
+           1. Let _fv_ be the substring of _sv_ from position _dp_ + 1, inclusive.
+           1. Let _f_ be ? ToNumber(_fv_).
+           1. Let _v_ be ? ToLength(? Get(_fv_, *"length"*)).
+         1. Let _i_ be ? ToNumber(_iv_).
```

Comment:
I'd have to do that for `n` as well. And I can't find any reference to `Math.abs` in ecma 262 or 402, so I wasn't sure if it's allowed.